### PR TITLE
Split tracer to a sub-package of katello-host-tools

### PR DESCRIPTION
**Pull request by ehelms**
For plugin updates, please indicate which repos this should be built into:

* [x] Nightly
* [ ] 1.16
* [ ] 1.15
* [ ] 1.14

See Foreman's [plugin maintainer documentation](http://projects.theforeman.org/projects/foreman/wiki/How_to_Create_a_Plugin#Release-strategies) for more information.

---

### CODE REVIEW

File: katello/katello-host-tools/katello-host-tools.spec [hunk 1]
```diff
@@ -160,18 +170,22 @@ exit 0
 /var/cache/katello-agent/
 %attr(750, root, root) %{_sbindir}/katello-package-upload
 %attr(750, root, root) %{_sbindir}/katello-enabled-repos-upload
-%{_prefix}/lib/yum-plugins
```
Comment (ekohl) [inline katello/katello-host-tools/katello-host-tools.spec]: Would it make sense to keep this but add an ```%exclude %{_prefix}/lib/yum-plugins/tracer_upload.py``` and would that even work?

Comment (ehelms) [reply katello/katello-host-tools/katello-host-tools.spec]: I honestly don't know without further testing.

Comment (jlsherrill) [reply katello/katello-host-tools/katello-host-tools.spec]: I tend to be more explicit with these things.  For example if i added a new plugin in a new subpackage, i'd have to remember to exclude it there. But I can see advantages to both

**Comment by jlsherrill** [comment]
LGTM, but jenkins didn't actually build anything for the tests.  I'd like to see what it builds first :)

**Comment by ekohl** [comment]
You're right:
```
15:19:54 Releasing to target: koji-katello
15:19:54 WARNING: unable to lookup latest package tag, building untagged test project
15:19:54 WARNING: .tito/packages/katello-host-tools doesn't exist in git, using current directory
15:19:54 Working in: /var/lib/workspace/workspace/packaging_build_rpm/rel-eng/build/release-katello-host-toolsN5KlPe
15:19:54 Building release in Koji...
15:19:54 WARNING: katello-host-tools not specified in whitelist for katello-nightly-rhel7
15:19:54    Package *NOT* submitted to Koji.
```

**Comment by ehelms** [comment]
[test]

**Comment by ehelms** [comment]
Good catch @jlsherrill the job wasn't handling this and the packaging was a bit off.

**Comment by ehelms** [comment]
@jlsherrill Please re-review as I had to make a few changes

File: comps/comps-katello-client-rhel7.xml [hunk 2]
```diff
@@ -15,6 +15,7 @@
        <packagereq type="default">katello-agent</packagereq>
        <packagereq type="default">katello-host-tools</packagereq>
        <packagereq type="default">katello-host-tools-fact-plugin</packagereq>
+       <packagereq type="default">katello-host-tracer</packagereq>
```
Comment (ehelms) [inline comps/comps-katello-client-rhel7.xml]: Oops -- found my own typo

**Pull request merged**